### PR TITLE
Add messages french translations

### PR DESCRIPTION
**Pull request by sdaoudi**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ $ php app/console ngadminui:install
 This will install all required configuration in two places:
 
 * `app/config/ngadminui.yml`
-* `ezpublish_legacy/siteaccess/NEW_SITEACCESS_NAME`
+* `ezpublish_legacy/settings/siteaccess/NEW_SITEACCESS_NAME`
```
Comment (emodric) [inline INSTALL.md]: Isn't this already merged? :)

You can rebase the pull request to get rid of this diff.

**Comment by emodric** [comment]
Thanks @sdaoudi :)

I've left a comment about a change which is already present in master, so when you resolve it, I can merge :)

File: Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+netgen_admin_ui.login.message.username_password: 'Vérifier votre nom d''utilisateur et votre mot de passe.'
+netgen_admin_ui.login.message.letter_case: 'Toutes les lettres doivent respecter la casse.'
+netgen_admin_ui.login.message.contact_admin: 'Merci de réessayer ou de contacter l''administrateur du site.'
+netgen_admin_ui.login.username: 'Nom d''utilisateur'
+netgen_admin_ui.login.username.title: 'Saisir un nom d''utilisateur valide dans ce champ.'
+netgen_admin_ui.login.password: 'Mot de passe'
+netgen_admin_ui.login.password.title: 'Saisir un mot de passe valide dans ce champ.'
+netgen_admin_ui.login.submit: 'Se connecter'
+netgen_admin_ui.login.submit.title: 'Click here to log in using the username/password combination entered in the fields above.'
```
Comment (emodric) [inline Resources/translations/messages.fr.yml]: Did you miss this one?

File: Resources/translations/messages.fr.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+netgen_admin_ui.user.on: 'Activé'
+netgen_admin_ui.user.off: 'Désactivé'
+netgen_admin_ui.user.logout: 'Se déconnecter'
+
+netgen_admin_ui.menu.tags: 'Netgen Tags'
+netgen_admin_ui.menu.cache: 'Cache'
+netgen_admin_ui.menu.clear_cache: 'Vider le cache'
+netgen_admin_ui.menu.bookmarks: 'Les signets'
+
+netgen_admin_ui.layouts.mappings.applied_to: 'Layouts applied to'
```
Comment (emodric) [inline Resources/translations/messages.fr.yml]: These four strings are also untranslated. Intentionally?

**Comment by sdaoudi** [comment]
@emodric, can you close this pull request please.

Il will create a new one with all translations.

Thanks :)

**Comment by emodric** [comment]
Alright, thanks :)

**Comment by sdaoudi** [comment]
Thank you @emodric.

Pull request created #5 :+1:

**Pull request closed without merging**